go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-core v1.1.19
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
 	github.com/ElrondNetwork/elrond-go-logger v1.0.7
-	github.com/ElrondNetwork/elrond-vm-common v1.3.14
+	github.com/ElrondNetwork/elrond-vm-common v1.3.15-0.20220825161922-b9b0bd5da6e0
```

Comment:
please create a proper  1.3.15 release

Reply:
used the proper release.
I cannot target it towards rc/2022-july since the branch cannot be tested without PR #4395 
The target branch will automatically be updated after it is merged. fyi @iulianpascalau